Hippo.Web/Controllers/SoftwareController.cs

```diff
@@ -0,0 +1,84 @@
+using Hippo.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Hippo.Core.Models;
+using Hippo.Email.Models;
+
+namespace Hippo.Web.Controllers;
+
+[Authorize]
+public class SoftwareController : SuperController
+{
+    private readonly IUserService _userService;
+    private readonly IHistoryService _historyService;
+    private readonly IEmailService _emailService;
+    private readonly INotificationService _notificationService;
+
+    public SoftwareController(IUserService userService, IHistoryService historyService,
+        IEmailService emailService, INotificationService notificationService)
+    {
+        _userService = userService;
+        _historyService = historyService;
+        _emailService = emailService;
+        _notificationService = notificationService;
+    }
+
+    [HttpPost]
+    public async Task<ActionResult> RequestInstall([FromBody] SoftwareRequestModel softwareRequestModel)
+    {
+        if (!ModelState.IsValid)
+        {
+            return BadRequest(ModelState);
+        }
+
+        var currentUser = await _userService.GetCurrentUser();
+
+        // generate plain text email that will be forwarded to ServiceNow
+        var emailModel = new EmailModel
+        {
+            Emails = new string[] { "hpc-help@ucdavis.edu" },
+            Subject = "HPC Software Install Request",
+            TextBody = @$"Category: Request
+
+Subcategory: New
+
+Caller: {currentUser.Kerberos}
```

Comment:
Name for IAM or an email

Reply:
Adding a Typeahead to search for user...